README.adoc

```diff
@@ -418,3 +420,22 @@ In response you should get ID of created item, using that you can retrieve the w
 ----
 $ ./bin/wit-cli show workitem --id <ID> -H localhost:8080 --pp
 ----
+
+== Prod preview setup
+
+In order to setup wit to work against link:https://prod-preview.openshift.io/[prod preview] one needs to update `token.publickey` in config.yaml.
+
+NOTE: Public key can be obtained from https://sso.prod-preview.openshift.io/auth/realms/fabric8/
+
+Also, the following environment variables must be set:
+
+* `export F8_DEVELOPER_MODE_ENABLED=1`
+* `export F8_OPENSHIFT_TENANT_MASTERURL="https://console.free-int.openshift.com/"`
+* `export F8_KEYCLOAK_REALM="fabric8"`
```

Comment:
@hectorj2f are you sure about "also" :-) ? I have spend a lot of time to setup wit against prod-preview & without a ll this instructions I was not able to make it work. Actually, those instructions are not che-starter specific (it is just about how to start wit against prod-preview) and I think prod-preview section is more relevant for those.

So, you think that  `export F8_KEYCLOAK_REALM="fabric8"` can be safely removed ?